### DISCUSSION TITLE
Akash Insiders, Vanguards and Community Support Proposal

### DISCUSSION THREAD

**Discussion by CaptainSplince** [akash-network/community#165] [Governance Proposals]
**Overview**
The Akash community is vast. It spans across geographic regions and social media channels. This is a good thing! The more people who follow Akash activities, the more likely they introduce new community members to the ecosystem. However, with the ever growing size of the Akash community, the ability for community support, engagement and education becomes difficult at scale. 

This proposal includes three current and future initiatives that are meant to support the Akash Community

**1. Insiders & Vanguards Program**

The Akash Insider’s program launched with much fanfare in late 2021. The initial concept of the program was to find community members who have either been actively engaged in Akash related activities or onboard community members who have shown interest in participating in Akash activities. We initially onboarded 10 Insiders. Since then it has grown to 50 across the globe. 

The Akash Vanguards program launched in mid-2022. Vanguards are the Akash community's most engaged members. It is a small group of highly active and supportive community members. Currently we have 9 Akash Vanguards 

**Insider Responsibilities**
There are no requirements to join the Insiders community. It is completely volunteer based with optional activities. That being said, there is one unwritten rule for joining the program. It is expected that if you apply and join, you will be fairly active in community engagements. Examples but not limited to - Share feedback, highlight issues, join calls, provide community support.

**Vanguard Responsibilities**
The Akash Vanguards program has requirements to join. However, requirements are incentivized. A full overview of the program can be [found here](https://docs.google.com/document/d/1IMOLARSNg5tnIsfQ4B5mfDrmx-MDJNpIFbLxFLuldas/edit).

**Insider Rewards**
Since there are no requirements to join the program, there are no standardized rewards, incentives or a salary for joining the program. However, there are ad hoc activities that are optional to get involved in that can have rewards attached. Ex: Testnet support, blog bounties, Akash feature updates, ect.

**Vanguard Rewards**
Since the Vanguard’s program has requirements attached to it, there are rewards given to members. Full details can be found on the [Vanguards Overview](https://docs.google.com/document/d/1IMOLARSNg5tnIsfQ4B5mfDrmx-MDJNpIFbLxFLuldas/edit).

**2. Community Bounties**

This proposal includes a fund specifically for community members that want to contribute to the Akash ecosystem through bounties. Bounties include any effort that contributes to the Akash ecosystem both technical and non technical. Bountis will be decided on through conversations happening in the various SIG and WG calls and discussions on Discord.

**3. Community Note Takers**

With the launch of the Akash community's open framework for development, there are now numerous ways and opportunities to contribute to Akash. To make contributions easier for the community to get involved with Special Interest groups and Working groups have been formed to encourage contributions. There groups have regular meetings where the community comes to consensus on how things should get accomplished. Each meeting is recorded for posterity, In addition to recording, notes are taken for those in the community that would like to review. Community members are incentivised to take notes. The budget included in this proposal includes rewards for note takers within the community. 

**2023 Community Budget** 
The full overview of community funds requested to support this program can be [found here](https://docs.google.com/spreadsheets/d/1NMB_-m9GPZEsg0vytgIjDzAPkJJGsiCV7HqixCNT-L4/edit?usp=sharing).

**2023 Community Budget TLDR:**
Insiders: $7,000 Paid in AKT
Vanguards: $120,000 paid in AKT
Community Rewards/Bounties/Support: $20,000 paid in AKT
SIG & WG Note Takers: $17,600 USD paid in AKT

**Quarterly Breakdown:**

Budget will be taken out and proposed quarterly in new governance proposals.. Each quarter budget will be reviewed and proposed given community fund resources and market conditions. Overclock labs will be the custodian of the funds. Any funds not used will be returned to the community pool.

Q2: $54,866
Q3: $54,866
Q4: $54,866

**Total Budget request:** $164,600 USD paid in AKT

_Next steps:_
After community feedback is given over the next few weeks, this proposal will go live for a funding vote.

**Comment by baktun14** [comment]
Hey Adam, so there's no more concern about draining the community pool I suppose? This is for approximately half of what will be left after both the current proposals for events and hackathon are passed.

Who are the current vanguards and who will be the next ones? Have they provided significant enough value since the inception of the program?

**Reply by baktun14** [reply]
Also, there's more budget directed to events despite having a current proposal for events. This seems redundant.

![image](https://user-images.githubusercontent.com/15185355/231319382-d15dc77c-ba07-4e5f-877a-34b3dd943ef8.png)

**Reply by ghost** [reply]
I understand payments will be made in a quarterly basis, so not all will be withdrawn at once? 

I guess what we should worry about is how is the community pool getting funds, however, that's a topic for another discussion 

About this proposal tho, the community needs to take care of Akash, I support the funding of committed and working community members 100% since these activities put Akash in front of new eyes, new investors, new builders, and overall more exposure to Akash brand,

How would you measure the value these folks bring to Akash Network?

**Reply by figurestudios** [reply]
> Hey Adam, so there's no more concern about draining the community pool I suppose? This is for approximately half of what will be left after both the current proposals for events and hackathon are passed.

I believe that this is a good concern. The Vanguards program is great, but the question to be asked is whether or not optimizations need to be made, and if so, in what way. $120k is as much as a full-time employee with overhead and the results from the program definitely surpass what a single full-time employee would be able to do.

The main question to be asked, in my opinion, is if 24/7 availability of support (the goal) is needed or if some time-zones are not as important to support based off where most of the users are from. Before the Vanguards program, the support in the channels was horrible from my personal experience -- I both asked my own questions to never get responses, and saw other questions go unanswered as well. 

> Who are the current vanguards and who will be the next ones? Have they provided significant enough value since the inception of the program?

The Vanguards can be found by looking at the assigned roles on Discord, consisting of the people in the picture below and more:

![image](https://user-images.githubusercontent.com/64747030/231834995-dca66a5a-136b-4fa8-b7c3-f4a19e736f6f.png)

The Vanguards are - per the program - creating content, spreading the word of Akash, providing support in the Discord server, and more. A few weeks ago, [this message](https://discord.com/channels/747885925232672829/786250602173693993/1088138171885297704) was went in the #akash-insiders channel to highlight some of the Vanguards.

Again, it is a good concern whether or not it is worth $120k/yr. If this includes the 24/7 support goal, I believe so. The 24/7 support goal should be met through the Vanguards program, but everything else could be seperated into other programs and proposals for making easier budgeting.

**Comment by CaptainSplince** [comment]
This is great feedback. Thanks all. - I'll try answering each question from my perspective.

#### 1) Concern about draining the community pool
>  The way this proposal is drafted is to highlight the full cost of the program over a 1 year period. However, Proposals will be made quarterly as not to "drain" the community pool. For example, once feedback is included in the proposal and the idea can move forward, there will only be 1 quarter of funding requested. Each quarter funding will be proposed again. 

#### 2) Who are the current vanguards and who will be the next ones? 
> You can find the Vanguards in Discord. They are all tagged with the "Vanguards" role

#### 3) Have they (Insiders/Vanguards) provided significant enough value since the inception of the program?
> As FigureStudios mentioned above, before the Vanguards program, community support was limited. Many community support questions would go unanswered which is not a great experience for new Akash users. As also mentioned, the primary goal of the Vanguards program is to provide 24/7 community support. eg - "the sun never sets on the Akash network." 

>**Currently, over the last 90 days, 8 of the 10 most engaged (eg - replying to questions) Discord members have been Vanguards according to data provided by Orbit**

>In addition to support, there are secondary goals of the Vanguards program, a kind of choose your own adventure way of community support and growth. You can view those in the detailed Vanguards program document linked above. 

#### 4) Event support budget ($4k)
> This is seperate from the 2023 Akash Event proposal because that proposal is specifically for getting involved in IRL events NOT owned by the Akash community. This proposal includes a small $4k budget is for Insiders or Vanguards who are looking for support to host their own local meetups and to build hyper local community. The budget is meant for swag, giveaways or catering for example.

Thanks again for the feedback and questions all.

**Comment by SophoclesSummer** [comment]
This proposal was approved on 7/14/2023 by a 90% majority. 
https://www.mintscan.io/akash/proposals/208
